Enhancing the documentation for the variables

DianaMariaDDM opened the pull request:
**Overall Review of Changes:**
The documentation of the variables is more detailed and improved.

**Issue Fixes:**
- https://github.com/ansible-lockdown/DEBIAN12-CIS/issues/35

**How has this been tested?:**
N/A

DianaMariaDDM committed:
First part of fixing the documentation for the variables process.

Signed-off-by: Diana-Maria Dumitru <diana.dumitru@siemens.com>

DianaMariaDDM committed:
Second part of fixing the documentation for the variables process.

Signed-off-by: Diana-Maria Dumitru <diana.dumitru@siemens.com>

DianaMariaDDM committed:
Fixing the documentation for the `deb12cis_remote_log_server` variable.

Signed-off-by: Diana-Maria Dumitru <diana.dumitru@siemens.com>

DianaMariaDDM committed:
Fixing documentation.

Signed-off-by: Diana-Maria Dumitru <diana.dumitru@siemens.com>

DianaMariaDDM committed:
Removing not used variables

Signed-off-by: Diana-Maria Dumitru <diana.dumitru@siemens.com>

DianaMariaDDM committed:
Update some variable documentation.

Signed-off-by: Diana-Maria Dumitru <diana.dumitru@siemens.com>

uk-bolly commented:
hi @DianaMariaDDM 

We are seeing merge conflicts on this PR. Im afraid if you can resolve am happy to merge.

Many thanks

uk-bolly

DianaMariaDDM committed:
Merge branch 'devel' into documentation_for_the_variables

Signed-off-by: DianaMariaDDM <143085010+DianaMariaDDM@users.noreply.github.com>

DianaMariaDDM committed:
Fixing lint problems.

Signed-off-by: Diana-Maria Dumitru <diana.dumitru@siemens.com>

DianaMariaDDM committed:
Merge branch 'documentation_for_the_variables' of code.siemens.com:infosec-pss-gov/security-crafter-baseline-automations/ansible-lockdown/debian12-cis into documentation_for_the_variables

DianaMariaDDM commented:
Hi @uk-bolly,

I fixed the conflicts. Everything should have worked fine, unfortunately the playbook tests fails [here](https://github.com/ansible-lockdown/DEBIAN12-CIS/actions/runs/13917177080/job/38942216575?pr=40#step:12:122), complaining about `deb12cis_selinux_disable` being undefined. Indeed it is not, we have deleted it as it was not used anywhere around the role and we figured it was not needed.

uk-bolly commented:
hi @DianaMariaDDM 

Thank you as always for your feedback and PRs. It seems we managed to cross over in work in the PR and i beleive the enhancements you have provided have been incorporated and a new version had been released in between.
I will therefore close this PR. 
Thank you once again

uk-bolly
